tests/test_cv.py

```diff
@@ -0,0 +1,19 @@
+import scanpy as sc
+
+from augurpy.cv import run_cross_validation
+from augurpy.estimators import create_estimator
+from augurpy.read_load import load
+
+adata = sc.read_h5ad("tests/sc_sim.h5ad")
```

Comment:
Think it might be cooler to use `setup_method` if we wrap this into a class. Like this: https://github.com/theislab/ehrapy/blob/development/tests/api/tools/nlp/test_translator.py#L23

Reply:
What is the advantage of wrapping it into a class like you did with TestDeepL vs just running functions?
Also, just so I understand this correctly: the start-up methods sets up self, which is then passed to all other methods as well right? This is why self is written everywhere?:)

Reply:
Then I would leave it for now, and come back to it when I revise the test cases - if thats okay.

Reply:
All right.